### PR TITLE
refactor: remove amazon license

### PR DESCRIPTION
**Pull request by Kavlahkaff**
Removed Amazon license from all .py files. 

Changed pre-commit hook, so it does not add the license again after each commit.

----

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by aaronkl — APPROVED** [review]
Looks good! Thanks for the PR

File: syne_tune/optimizer/schedulers/searchers/hypertune/hypertune_bracket_distribution.py [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# A copy of the License is located at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-import numpy as np
```
Comment (aaronkl) [inline syne_tune/optimizer/schedulers/searchers/hypertune/hypertune_bracket_distribution.py:12]: Could you put this import back?

File: tst/blackbox_repository/test_simulated_backend.py [hunk 2]
```diff
@@ -1,15 +1,3 @@
...
-# Licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License.
-# A copy of the License is located at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-import numpy as np
```
Comment (aaronkl) [inline tst/blackbox_repository/test_simulated_backend.py:12]: Could you put this import back?

Comment (Kavlahkaff) [reply tst/blackbox_repository/test_simulated_backend.py:12]: Missing imports are put back in, this should work now

File: githooks/pre-commit [hunk 3]
```diff
@@ -8,12 +8,3 @@ if [ $? -ne 0 ]; then
  exit 1
 fi
 
-
-echo "Adding license header."
-./license-add.sh
```
Comment (geoalgo) [inline githooks/pre-commit:13]: We probably want to remove this file as well.

Comment (Kavlahkaff) [reply githooks/pre-commit:13]: Hi, thanks for the notice. I removed it now.

**Review by aaronkl — APPROVED** [review]

**Pull request merged**